backend/build.gradle.kts

```diff
@@ -49,9 +49,14 @@ dependencies {
     implementation("org.jetbrains.exposed", "exposed-core", exposed_version)
     implementation("org.jetbrains.exposed", "exposed-dao", exposed_version)
     implementation("org.jetbrains.exposed", "exposed-jdbc", exposed_version)
-    implementation("com.impossibl.pgjdbc-ng", "pgjdbc-ng", "0.8.6")
+    implementation("org.postgresql", "postgresql", "42.2.18")
```

Comment:
Why did you remove the impossibl?

Reply:
Has no implementation for postgis columns and they don't plan to implement it. The main postgres jdbc driver worked straight away.

Does the impossibl driver have any advantages we use/wanted to use? We could maybe use different drivers for data importing and querying but idk if that is a good idea